### DISCUSSION TITLE
Tuần 9 nè các bạn ơi!!

### DISCUSSION THREAD

**Discussion by TranBaoLam10C13** [dangxuantruong10a13/nhom9_10a13#17] [General]
tui làm câu 1,2 bài 5 r đó ông trường làm pwp thôi
còn 2 câu bài 6 lan anh vs hân chia nhau làm nha

**Comment by dangxuantruong10a13** [comment]
Oke lun anh pn, mà là làm pwp hả chứ kp work giống bài trc à

**Reply by TranBaoLam10C13** [reply]
Lm giống bữa trc lú

**Reply by dangxuantruong10a13** [reply]
Word;)

**Reply by TranBaoLam10C13** [reply]
Uk "

**Reply by TranBaoLam10C13** [reply]
Đăng lên code r

**Reply by dangxuantruong10a13** [reply]
Mà powerpoint cần thuyết trình nữa hay sao, chứ bài này để thầy đọc thôi mà đk

**Reply by TranBaoLam10C13** [reply]
Ko làm nộp thôi

**Reply by dangxuantruong10a13** [reply]
Hay là làm word típ;))) mình làm tài lịu mà 🤔

**Reply by dangxuantruong10a13** [reply]
Tui sợ pp nhiều chữ quá hết zjp

**Reply by dangxuantruong10a13** [reply]
Chia câu 2 tui làm nữa

**Reply by TranBaoLam10C13** [reply]
Word như bth

**Reply by dangxuantruong10a13** [reply]
🥴🥴

**Reply by TranBaoLam10C13** [reply]
Ông làm word dc r ông nhiều việc mà :vv

**Reply by dangxuantruong10a13** [reply]
Oke z word giống trc mình làm hay common :v

**Reply by TranBaoLam10C13** [reply]
Trang trí nx

**Reply by dangxuantruong10a13** [reply]
Oke z làm word tui tự trang trí đồ nha

**Reply by TranBaoLam10C13** [reply]
Uk

**Reply by dangxuantruong10a13** [reply]
Oke zậy đi @04lananh @NgHan12pn

**Reply by 04lananh** [reply]
tui với Hân chia nhau rùi nha 
t làm câu 4 đó

**Reply by NgHan12pn** [reply]
khi nào tui làm xong câu 3 thì tui gửi vd cho bà làm câu 4 nha @04lananh

**Reply by NgHan12pn** [reply]
ủa mng làm xong có cần ghi nguồn phía cuối không ??

**Reply by TranBaoLam10C13** [reply]
Có quên nx

**Reply by NgHan12pn** [reply]
Ê mà cái câu này t tự làm nên là hong có nguồn j hết á

**Reply by HienCaoPhuNhuan** [reply]
> Oke lun anh pn, mà là làm pwp hả chứ kp work giống bài trc à

Mình không cần làm ppt

**Reply by dangxuantruong10a13** [reply]
Dạ cám ơn thầy

**Comment by TranBaoLam10C13** [comment]
https://toploigiai.vn/ly-thuyet-tin-hoc-10-bai-5-ngon-ngu-lap-trinh
https://magenest.com/vi/ngon-ngu-lap-trinh-bac-cao/
Nguồn bài tui nha trường

**Comment by 04lananh** [comment]
Ủa mng, có 5 cái vd là giờ tui ngồi làm hết cả 5 luôn hả (liệt kê với sơ đồ khối ) hay là chỉ làm vài cái thôi

**Reply by HienCaoPhuNhuan** [reply]
> Ủa mng, có 5 cái vd là giờ tui ngồi làm hết cả 5 luôn hả (liệt kê với sơ đồ khối ) hay là chỉ làm vài cái thôi

con chọn ra 3 cái con tự tin nhất và tự hào nhất. Nếu có thể thì làm hết cũng được

**Reply by 04lananh** [reply]
Dạ thầy ơi làm 3 cái xong còn phải viết chương trình nữa hả thầy

**Reply by HienCaoPhuNhuan** [reply]
không, con chỉ viết thuật toán bằng liệt kê bước và sơ đồ khối thôi. Mấy đứa đã học code đâu

**Reply by 04lananh** [reply]
dạ con cảm ơn ạ

**Reply by dangxuantruong10a13** [reply]
Hình như có chữ nếu không với hoặc mà đk. Chắc là 3 bài toán hoặc 2 vd thôi à

**Reply by dangxuantruong10a13** [reply]
Phải hok 🤔

**Reply by NgHan12pn** [reply]
Thì @04lananh chọn ra 3 cái nào mà cảm thấy dễ với mình thì làm thôi

**Reply by dangxuantruong10a13** [reply]
Mà tui cũng k biết cái chữ nếu không là liên kết với câu hỏi hay là cho 2 ví dụ nựa

**Reply by dangxuantruong10a13** [reply]
Nếu cho 2 ví dụ chắc là chữ hoặc ở dưới bà có thể chọn bài toán hoặc ví dụ

**Reply by dangxuantruong10a13** [reply]
3 hoặc 2

**Reply by 04lananh** [reply]
ủa mng cho hỏi 1 câu ngoài lề đc ko?
sao tên tui lại có màu nữa vậy

**Reply by NgHan12pn** [reply]
Là sao ???

**Comment by dangxuantruong10a13** [comment]
Mình đã tổng hợp và trang trí xong, mn vào xem trong phần code
Cách dùng sơ đồ khối của bạn Lan Anh cần vẽ ra luôn k lúc trước thầy có cân nhắc, mn thấy sao, giờ cũng trễ r

**Reply by dangxuantruong10a13** [reply]
nếu làm thì deadline 9h hôm nay để mình nộp nữa, ai giúp bạn Lan Anh làm lại sơ đồ khối k?

**Reply by TranBaoLam10C13** [reply]
Phần code chưa thấy

**Reply by dangxuantruong10a13** [reply]
Up r đoá

**Reply by NgHan12pn** [reply]
Ủa tưởng hạn nộp là ngày hqua r chứ

**Reply by HienCaoPhuNhuan** [reply]
> Ủa tưởng hạn nộp là ngày hqua r chứ

Hôm nay chứ con

**Reply by NgHan12pn** [reply]
À dạ con nhầm ngày,sin lũi thầy "hiền" nhìu ạ

**Reply by dangxuantruong10a13** [reply]
Rầu có ý kiến j hok D;

**Reply by NgHan12pn** [reply]
Hmm chắc cũng ổn á

**Reply by NgHan12pn** [reply]
Mà bây giờ vẽ sơ đồ khối thì kh kịp đâu

**Reply by dangxuantruong10a13** [reply]
Thâu đc rầu để t chơi lun
